src/components/app/MenuButtons/index.css

```diff
@@ -52,6 +52,13 @@
   content: '';
 }
 
+/* position: relative so that we can use position: absolute for the generated
+ * element below. */
+.menuButtonsButton-hasRightBorder,
+.menuButtonsButton-hasLeftBorder {
+  position: relative;
+}
```

Comment:
I feel like we don't need those anymore :)

Reply:
Yeah but I believe this is still more correct. Without these we rely on the fact that `menuButtonsButton` has `position: relative` which is a bit fragile. So I'd rather keep it if you don't mind :-)